packages/web-app/src/controllers/questionnaire-check-and-confirm.js

```diff
@@ -0,0 +1,147 @@
+const reviewOutcome = {
+  COMPLETE: 'COMPLETE',
+  INCOMPLETE: 'INCOMPLETE',
+};
+
+const documents = ['Passport', 'Birth certificate'];
+
+const siteAddress = {
+  address1: 'Jaleno',
+  address2: 'Church Lane',
+  town: 'Eakring',
+  city: 'NEWARK',
+  postcode: 'NG22 ODH',
+};
+
+const compileAddress = (address) => {
+  const mappedAddress = Object.values(address).map((item) => `${item} <br> `);
+  return mappedAddress.toString().replace(/,/g, '');
+};
+
+const compileMissingDocuments = (items) => {
+  const mappedDocuments = items.map((item) => `${item} <br>`);
+  return mappedDocuments.toString().replace(/,/g, '');
+};
+
+const getConfirmationSections = (appealReference, outcome) => {
```

Comment:
As of right now it isn't needed but the purpose of these functions is to guide development when we have the ability to interface with the db, basically trying to front load as much development as possible within this frontend ticket.

Reply:
This `appealReference` param is populated from the `appealId` param passed in the URL and the appeal id isn't going to be displayed on this page. The appeal reference however is and this can be passed in with the appeal data, currently in `req.session`. This can also be used to provide the Local planning department value.

For now and until we connect the UI to the db, we have mock data in https://github.com/Planning-Inspectorate/back-office/blob/main/packages/web-app/src/lib/api-wrapper.js as this will probably be the function that fetches the data and the idea is that this can be used to provide data to the app and by using the mock data and keeping the pages consistent we should be able to build the app mostly as we would like then hopefully only have to make minimal changes once the actual data is available.

If necessary this can be refactored when we have the actual data available.

Reply:
I think that's the essence of what I'm saying, we can change it but it will be refactored again once we have the data available so is it worth doing in this PR when the aim is to get the frontend ready. 

Also are we saying that the appeal data is going to be readily available in req.session? Because that wasn't apparent when working on the ticket which is why the appealReference param is there in the first place.